### ISSUE TITLE
there is some issues with regard to understanding the experimental

### ISSUE THREAD

**Issue by gabriben** [gabriben/multilabel#75]
effect of the proposed loss function in the context of pre-trained models that
use a different loss function
